Dockerfile

```diff
@@ -61,3 +61,5 @@ RUN export INSTALLED_KERNEL=$(rpm -q --qf "%{VERSION}-%{RELEASE}.%{ARCH}"  kerne
     export INSTALLED_RT_KERNEL=$(rpm -q --qf "%{VERSION}-%{RELEASE}.%{ARCH}"  kernel-rt-core); \
     echo "{ \"KERNEL_VERSION\": \"${INSTALLED_KERNEL}\", \"RT_KERNEL_VERSION\": \"${INSTALLED_RT_KERNEL}\", \"RHEL_VERSION\": \"${RHEL_VERSION}\" }" > /etc/driver-toolkit-release.json
 
+LABEL com.coreos.rpm.kernel="${INSTALLED_VERSION}" \
```

Comment:
Thanks for that PR @fabiendupont.
I have 2 questions:
1. Where does those variables come from `INSTALLED_VERSION` and `INSTALLED_RT_VERSION`?
2. How do you plan to consume those variables? Eventually, in order to get the manifest, you will need to query a specific tag no?! How are you planing to iterate over those tags?